### DISCUSSION TITLE
Welcome to openvslam Discussions!

### DISCUSSION THREAD

**Discussion by ymd-stella** [stella-cv/stella_vslam#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by surfii3z** [comment]
Hi, my name is Surf. Looking forward to see this community grows :)
